test/journeys/specs/oneOnOne/messaging.js

```diff
@@ -112,5 +112,184 @@ describe(`Widget Space: One on One`, () => {
 
     it(`sends and flags message`);
 
+    describe(`markdown messages`, () => {
+      it(`sends message with bold text`, () => {
+        // Increase wait timeout for message delivery
+        browser.timeouts(`implicit`, 10000);
+        browserLocal.waitForVisible(`[placeholder="Send a message to ${mccoy.displayName}"]`);
+        browserRemote.waitForVisible(`[placeholder="Send a message to ${spock.displayName}"]`);
```

Comment:
You could probably set these three lines in a `beforeEach` within this describe block to slim down your "it" blocks.